packages/renderer/src/docs/datahub.vue

```diff
@@ -1,3 +1,138 @@
 <template>
-  <h5>DataHub Help</h5>
-</template>
+  <div>
+    <h5>Download ARC</h5>
+    <div class="q-pa-sm text-body2 text-justify">
+      In the "Download ARC" menu, you can download ARCs from the DataHUB.
+    </div>
+
+    <div class="q-pt-sm text-justify">
+      <q-list>
+
+        <q-item>
+          <q-item-section>
+            <div class="col">
+              <div
+                class="q-field row no-wrap q-field--outlined q-field--labeled q-field--dense col-grow q-field--outlined q-field__inner relative-position col self-stretch">
+                <div class="q-field__control relative-position row no-wrap">
+                  <div class="col relative-position row">
+                    <input class="q-field__native q-placeholder">
+                    <div class="q-field__label absolute">ARC Identifier</div>
+                  </div>
+                  <div class="q-field__append q-field__marginal row no-wrap items-center">
+                    <i class="q-icon notranslate material-icons">search</i>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+          </q-item-section>
+          <q-item-section>
+            Use the search field to find an ARC by the name of the ARC or its creator.
+          </q-item-section>
+        </q-item>
+
+        <q-item>
+          <q-item-section>
+            <div class="col">
+              <div class="row items-center q-checkbox--dense">
+                <div class="q-checkbox__inner relative-position">
+                  <div class="q-checkbox__bg absolute"></div>
+                </div>
+                <div class="q-checkbox__label">LFS</div>
+              </div>
+            </div>
+
+          </q-item-section>
+          <q-item-section>
+            (Un)check the LFS box to (not) download LFS objects
```

Comment:
Only abbrevation, this does not really explain what LFS does or what it means if the box is ticked or not